### ISSUE TITLE
What goes in scriptexec.cfg

### ISSUE THREAD

**Issue by samersultan** [valleyman86/EasyDMGReportGO#1]
The contents of CS_ReportDmg.ahk get pasted into scriptexec.cfg? 

I'm not clear on that step.

**Comment by valleyman86** [comment]
Nope. Just add the one line in autoexec and leave scriptexec.cfg blank

On Thu, Jan 2, 2014 at 9:30 PM, Samer Sultan notifications@github.comwrote:

> The contents of CS_ReportDmg.ahk get pasted into scriptexec.cfg?
> 
> I'm not clear on that step.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/valleyman86/EasyDMGReportGO/issues/1
> .

## 

_Joey Gentry_
_MurderDev http://www.MurderDev.com - Developing Killer Software_
